app/models/recurring_donation.rb

```diff
@@ -15,7 +17,8 @@ class RecurringDonation < ActiveRecord::Base
     :cancelled_at, # datetime of user/supporter who made the cancellation
     :donation_id, :donation,
     :nonprofit_id, :nonprofit,
-    :supporter_id #used because things are messed up in the datamodel
+    :supporter_id, #used because things are messed up in the datamodel
+    :anonymous
```

Comment:
Was there a particular reason you needed to add the `:anonymous`?

Reply:
I was getting a `Can't mass-assign protected attributes`

Reply:
Ah, alright.